cmd/cliconfig.go

```diff
@@ -14,7 +14,6 @@ type accountEntry struct {
 	Name   string `yaml:"name"`
 	UserID string `yaml:"userId"`
 	Token  string `yaml:"token"`
-	URL    string `yaml:"url"`
```

Comment:
As with the change in the make-config subcommand, what's the reason for this change?

Reply:
Added again.
Yes because I used another url, had to do some changes to test the build